zerver/webhooks/flock/doc.md

```diff
@@ -0,0 +1,32 @@
+# Flock webhook
+
+Get Zulip notifications from your Flock channels.
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+Login into to your [Flock](https://flock.com) account, follow below
+steps given below to get started.
+
```

Comment:
Delete the above two lines, you can always assume that the user is logged in. :)